src/main/util.ts

```diff
@@ -16,3 +17,29 @@ if (process.env.NODE_ENV === 'development') {
     return `file://${path.resolve(__dirname, '../renderer/', htmlFileName)}`;
   };
 }
+
+export const padZeros = (num: number, len: number) => {
+  return String(num).padStart(len, '0');
+};
+
+export const integerDivide = (a: number, b: number) => {
+  return Math.floor(a / b);
+};
+
+export const writeFile = (filePath: string, data: any) => {
+  try {
+    writeFileSync(filePath, data, { flag: 'w' });
+  } catch (err) {
+    console.error(err);
```

Comment:
Hm, where is this being used and would it be better to have that function throw? We shouldn't normally get errors writing a file and if we do we definitely don't want to continue like nothing happened

Reply:
I reckon Patrick is right. Fail fast. 
I have added a ticket to our team backlog to display an error to the user when export fails.